### ISSUE TITLE
JsonSchema -> RAML incorrect

### ISSUE THREAD

**Issue by andyws** [raml-org/ramldt2jsonschema#14]
Docs state: 

> Limitations
> in js2dt,
> 
> the following JSON schema properties are not supported and as a result, will not be converted:
> format, title, not

However the following jsonschema generates the below raml including the not supported "title", "format" etal

```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "title": "Basis period",
  "type": "object",
  "properties": {
    "start": {
      "title": "Start date",
      "description": "Date your basis period began",
      "type": "string",
      "format": "date"
    },
    "end": {
      "title": "End date",
      "description": "Date your basis period ended",
      "type": "string",
      "format": "date"
    }
  },
  "required": [
    "start",
    "end"
  ],
  "additionalProperties": false
}

```
```
#%RAML 1.0 Library

types:
  BasisPeriod:
    title: Basis period
    type: object
    properties:
      start:
        title: Start date
        description: Date your basis period began
        type: string
        format: date
        required: true
      end:
        title: End date
        description: Date your basis period ended
        type: string
        format: date
        required: true
    additionalProperties: false

```

**Comment by brevity** [comment]
This should be resolved by PR #16

**Comment by brevity** [comment]
This Issue should be resolved by PR #16

**Comment by jstoiko** [comment]
Hi @andyws I tested again with the latest beta and I think it addresses your issue. The output of that same JSON schema is now:
```raml
#%RAML 1.0 Library

types:
  BasisPeriod:
    type: object
    properties:
      start:
        description: Date your basis period began
        type: string
        displayName: Start date
        required: true
        pattern: date
      end:
        description: Date your basis period ended
        type: string
        displayName: End date
        required: true
        pattern: date
    additionalProperties: false
    displayName: Basis period
```
If you want to check for yourself, you can download the beta:
```bash
$ npm install -g ramldt2jsonschema@beta
```

Let me know if you get a chance to look at this before I can merge & release.
